app/src/main/java/org/mozilla/focus/widget/BrowserNavigationOverlay.kt

```diff
@@ -27,11 +27,15 @@ enum class NavigationEvent {
             R.id.navButtonHome -> HOME
             R.id.navButtonSettings -> SETTINGS
             R.id.turboButton -> TURBO
+            R.id.pinButton -> PIN_ACTION
             else -> null
         }
     }
 }
 
+const val VAL_CHECKED = "checked"
```

Comment:
nit: add these add to the companion object of NavigationEvent, otherwise they'll end up in the autocomplete for all files (which isn't terrible, but not ideal).